W-18564689 a2a schema validation policy mb (#862)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/policies-included-a2a-schema-validation.adoc

The A2A Schema Validation policy validates that every request to the A2A server complies with the A2A schema. If validation fails it will report a Policy Violation.

There are no configuration parameters for this policy.

include::partial$policy-title-headers.adoc[tag=ui]

> The A2A Schema Validation policy is not supported in Local Mode.

Reviewer: ```suggestion
The A2A Schema Validation policy isn't supported in Local Mode.
```
